bot/ai_utils.py

```
@@ -4,15 +4,16 @@
 import logging
 
 import tiktoken
+
+from klog import *
```

Ruff (0.8.0)

`from klog import *` used; unable to detect undefined names

(F403)